clickhouse_options.go

```diff
@@ -56,6 +56,14 @@ const (
 	CompressionDeflate = CompressionMethod(0x96)
```

Comment:
nit: you can use github.com/dmarkham/enumer 

```go
//go:generate go run github.com/dmarkham/enumer -type CompressionMethod -trimprefix Compression --transform lower
```

Reply:
this will require something like this:
```go
//go:generate go run github.com/dmarkham/enumer -transform lower -type CompressionMethod -trimprefix Compression -output clickhouse_options_enum.go

type CompressionMethod byte

func (c CompressionMethod) String() string {
	switch c {
	case CompressionNone:
		return "none"
	case CompressionZSTD:
		return "zstd"
	case CompressionLZ4:
		return "lz4"
	case CompressionGZIP:
		return "gzip"
	case CompressionDeflate:
		return "deflate"
	default:
		return ""
	}
}

const (
	CompressionNone    CompressionMethod = CompressionMethod(compress.None)
	CompressionLZ4     CompressionMethod = CompressionMethod(compress.LZ4)
	CompressionZSTD    CompressionMethod = CompressionMethod(compress.ZSTD)
	CompressionGZIP    CompressionMethod = CompressionMethod(0x95)
	CompressionDeflate CompressionMethod = CompressionMethod(0x96)
)
```

probably not worth it